src/main/charts/jira/values.yaml

```diff
@@ -384,10 +384,15 @@ jira:
   # https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
   # Do not populate when deploying to OpenShift, unless anyuid policy is attached to a service account.
   #
+
+  # -- Whether to apply security context to pod.
+  #
+  securityContextEnabled: true
```

Comment:
Im thinking, would it make more sense to rename this property to `enabled` and move it into the `securityContext` stanza itself? This would then also follow the current pattern that is used, for instance; `ingress.create` & `nfsPermissionFixer.enabled` etc.

Reply:
I wanted to do that in first place, but that already exists and for my purpose I can't use it. The if logic currently implemented will force the default value to show up, see the code:
https://github.com/atlassian/data-center-helm-charts/blob/main/src/main/charts/jira/templates/statefulset.yaml#L29-L37

If I remember correctly, it was the else part (lines 32 and 33) that blocked me from reworking this in order to keep the backward compatibility. Though it may be okay for you to change from `else` to `else if .enabled`. I don't really know what is the use case here, plus it is deprecated, so I basically decided to stay away from it.

If you decide to change the `else` into `else if .enabled`, then also the `if not .fsGroup` have to be handled properly, but that seems easy.

Reply:
@in-journey apologies for the late response on this. Let me take a look today.